domains/Dockerfile

```diff
@@ -1,13 +1,17 @@
-# Use the official .NET SDK image
-FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0-noble AS build
 
-# Set the working directory
 WORKDIR /app
 
 COPY domains/authorization domains/authorization
 COPY domains/certificates domains/certificates
 
-# Run the certificates project with --swagger
 RUN dotnet run --project domains/certificates/Query.API/API/API.csproj --swagger
-# Run the proxy project with --swagger
+
 RUN dotnet run --project domains/authorization/Proxy/Proxy.csproj --swagger
+
+FROM mcr.microsoft.com/dotnet/runtime-deps:8.0-noble-chiseled AS final
+
+WORKDIR /swaggerfiles
```

Comment:
Why this change? You will have to change it in EO base as well

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1381/commits/d64112003eb8f3ea93f7a1df279f779d11ca677b